src/seedu/addressbook/common/Messages.java

```diff
@@ -15,4 +15,6 @@
             "java seedu.addressbook.Main [STORAGE_FILE_PATH]";
     public static final String MESSAGE_WELCOME = "Welcome to your Address Book!";
     public static final String MESSAGE_USING_STORAGE_FILE = "Using storage file : %1$s";
+    public static final String MESSAGE_STORAGE_ERROR = "Error while saving the file (Ensure the file is not given read only access";
```

Comment:
Good to see you have moved the message to the separate class, where it belongs. :+1: